Introductory Tests/mesh_utility.py

```diff
@@ -0,0 +1,170 @@
+    FORTRAN implementation by Tsoulis requires as input.
+    Args:
+        path (str):  the path where to write
+        nodes (list): list of vertices (cartesian coordinates)
+        faces (list): list of faces (triangles consisting of vertices' indices)
+
+    Returns:
+        None
+    """
+    with open(path + "topoaut", "w") as f:
+        for fac in faces:
+            f.write(" {} {} {}\n".format(fac[0] + 1, fac[1] + 1, fac[2] + 1))
+    with open(path + "xyzposnew", "w") as f:
+        for n in nodes:
+            f.write(" {} {} {}\n".format(n[0], n[1], n[2]))
+    with open(path + "dataut", "w") as f:
+        for fac in faces:
+            f.write(" 3")
+
+
+def write_other(path, name, nodes, faces):
+    """
+    Writes other files like .ply files.
+    Args:
+        path (str):  the path where to write
+        name (str): the name of the file to be created, the suffix determines the concrete write operation
+        nodes (list): list of vertices (cartesian coordinates)
+        faces (list): list of faces (triangles consisting of vertices' indices)
+
+    Returns:
+        None
+    """
+    mesh = om.TriMesh()
+    mesh.add_vertices(nodes)
+    mesh.add_faces(faces)
+    om.write_mesh((path+name), mesh)
+    # Alternative Framework - Code
+    # cells = [("triangle", faces)]
+    # mesh = mio.Mesh(nodes, cells)
+    # mesh.write((path + name))
+
+
+def create_mesh():
+
+    path = str(pathlib.Path("Introductory Tests").parent.resolve())
+
+    # Read the input .pk file
+    #print("Reading file...")
+    mesh_points, mesh_triangles = read_pk_file(path + "/3dmeshes/churyumov-gerasimenko.pk")
+
+    # Un-normalizing the scale
+    #   Conversion factor [to metric meters]: 3126.6064453124995
+    mesh_points = mesh_points*float(3126.6064453124995)
+    #print("Physical dimension along x (UN-normalized): ", max(mesh_points[:,0]) - min(mesh_points[:,0]), "Km")
+
+    # Simple rotatation of comet in plot
+    #mesh_points = mesh_points*-1
+
+    # Tetrahralize the mesh
+    #print("Tetrahralize...")
+    tgen = tetgen.TetGen(mesh_points, mesh_triangles)
+    nodes, elems = tgen.tetrahedralize()
+
+    # Print a vtk file, diplayable with ParaView
+    #print("Writing vtk files...")
+    tgen.write(path + "/mesh/67P/churyumov-gerasimenko.vtk")
+
+    # Print the input files for the C++ implementation of the Polyhedral Model
+    #print("Writing polyhedron-cpp input files..")
+    write_to_node_faces_ele_file(path + "/mesh/67P/", "churyumov-gerasimenko", nodes, mesh_triangles, elems)
+
+    # Print the input files for the FORTRAN implementation of the Polyhedral Model by Tsoulis
+    #print("Writing FORTRAN files..")
+    write__tsoulis_fortran_files(path + "/mesh/67P/", mesh_points, mesh_triangles)
+
+    # Write other files
+    #print("Writing other files...")
+    write_other(path + "/mesh/67P/", "churyumov-gerasimenko.ply", nodes, mesh_triangles)
```

Comment:
I don't think you actually need these files do you? If not then you can also remove the functions above that write them